spec/support/factory_girl.rb

```diff
@@ -6,6 +6,9 @@
     password 'password'
     password_confirmation 'password'
     sequence(:username) { |n| "user#{n}" }
+    profile_photo { Rack::Test::UploadedFile.new(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.